docs/source/_static/data_formatting_examples/exec_format_structured_data.py

```diff
@@ -0,0 +1,151 @@
+"""
+Format Structured Data
+======================
+
+Format a dataset where the filenames of the tiles provide most of the metadata required to
+organize them into the 5D set (image_type, fov, round, ch, zplane) of 2D tiles  (y, x).  The
```

Comment:
I suggest providing some common options for `image_type` here.

Reply:
Will do.